.github/workflows/runtests.yml

```diff
@@ -431,7 +431,7 @@ jobs:
       - uses: actions/setup-python@v4
         with:
           python-version: '3.11'
-      - run: pip install asv virtualenv
+      - run: pip install 'asv==0.5.1' virtualenv
```

Comment:
```suggestion
      - run: pip install virtualenv packaging asv
```

Reply:
This should work.

Reply:
Thanks for the hint. [This comment](https://github.com/airspeed-velocity/asv/pull/1324#issuecomment-1687429941) suggest we might be able to use plugin syntax so I'm trying that first as that'd be more robust.

Reply:
Unfortunately that's not a supported group yet. Though it should be present in 0.6.1